src/bp-forums/notifications.php

```diff
@@ -159,6 +159,10 @@ function bbp_format_buddypress_notifications( $action, $item_id, $secondary_item
 
 		return $return;
 	}
+
+	do_action( 'bbp_format_buddypress_notifications', $action, $item_id, $secondary_item_id, $total_items );
+
+	return $action;
```

Comment:
@krunal-buddyboss How is it related to bbPress or forums here? I don't understand. We can just return `$action` from here and it will be fine.

```suggestion
	return $action;
```